thruster_interface/scripts/thruster_interface_node.py

```diff
@@ -48,7 +55,17 @@ def output_to_zero(self):
                 pwm_msg.positive_width_us.append(neutral_pulse_width)
             self.pub_pwm.publish(pwm_msg)
 
+    def heartbeat_callback(self, msg):
+        self.prev_heartbeat = msg.stamp
+
     def callback(self, msg):
+        if (rospy.get_rostime() - self.prev_heartbeat).to_sec() > 1.0:
```

Comment:
This check will only occur if we still receive `thruster_forces` messages. If allocator is dead, we'll never find out.